src/components/dashboardV2/dashboard.js

```diff
@@ -76,6 +98,11 @@ class Dashboard extends React.Component {
 
     return (
       <React.Fragment>
+        <Onboarding
+          slides={this.getOnboardingSlides()}
+          ctaLabel={t('Got it, thanks!')}
```

Comment:
Will be possible to rename this variable, if not now maybe you can put a note just to remember the cahngem but we should rename the parameter **ctaLabel** to something easier to understand, like finalButtonActionLabel or finalActionLabel